docs/monitoring.rst

```diff
@@ -98,10 +112,21 @@ Reader Instances
 Cronjobs
 ''''''''
 
+Via log explorer:
 ::
 
     labels."k8s-pod/app_kubernetes_io/component"=~"^cron-.*$"
```

Comment:
Should it be `cron-<my-github-repository-name>` here and down in the query? 

I opened this to make sure they all match https://github.com/mozilla-it/webservices-infra/pull/6028